src/components/ShellView/ResultGraph.vue

```diff
@@ -49,7 +59,14 @@
       <div v-else>
         <h5>Overview</h5>
         <div v-if="counters.total.node > 0">
-          <p>Showing {{ counters.total.node }} nodes</p>
+          <p>Showing {{ counters.total.node }} nodes ({{ counters.total.node - hiddenNodes.length}} visible, {{hiddenNodes.length}} hidden)</p>
```

Comment:
If no node is hidden, you should not show `{{ counters.total.node - hiddenNodes.length}} visible, {{hiddenNodes.length}} hidden`.

Also, the wording does not make sense, it should be something like `Showing x/y nodes (z nodes are hidden)`

Reply:
When some nodes are hidden, you should also count hidden edges and show something similar for edge count.